container/pull.bzl

```diff
@@ -71,7 +71,11 @@ container_import(
             tag=repository_ctx.attr.tag)
     ]
 
-  result = repository_ctx.execute(args)
+  timeout = None
+  if "TIME_OUT" in repository_ctx.os.environ:
```

Comment:
Can we change this to `PULLER_TIMEOUT`, to make this a bit more self-documenting?